holidays/financial/brasil_bolsa_balcao.py

```diff
@@ -70,7 +70,7 @@ def _populate_public_holidays(self):
             self._add_holiday_apr_21(tr("Tiradentes"))
 
         if self._year >= 1925:
-            # Workers' Day.
+            # Worker's Day.
```

Comment:
But not here :) ?

Reply:
This Brazilian holiday is originally named "Dia do Trabalhador", and translated as "Worker's Day" in [Brazil holidays](https://github.com/vacanza/holidays/blob/d55c5b4d7b126c1f493e86d9303f3fc6404bff51/holidays/countries/brazil.py#L119). "Workers' Day" would be "Dia dos Trabalhadores".

Reply:
Maybe it's more about localization vs translation? They can use their own name for that but as long as it's May 1st I treat it as "International Workers' Day". 

It also can be a wrong opinion :) so we should find a solution in a discussion. Probably, something to think about later.